app/models/repository.rb

```diff
@@ -22,7 +22,8 @@ class Repository < RepositoryBase
             length: { maximum: Constants::NAME_MAX_LENGTH }
 
   scope :accessible_by_teams, lambda { |teams|
-    left_outer_joins(:team_repositories)
+    with_archived
```

Comment:
Do we need really it here? We are using it to get an active repository scope. Check:
`at_who_controller.rb`
`my_module_repositories_controller.rb`

Reply:
Hmm... You are correct. Based on the name, I would say it should include archived, but it's better to explicitly call `.with_archived` where we need them.